.devcontainer/Dockerfile

```diff
@@ -8,6 +8,6 @@ RUN install-apt make g++
 RUN install-tool python 3.11.4
 
 # renovate: datasource=npm
-RUN install-tool yarn 1.22.19
+RUN install-tool pnpm 8.6.11
```

Comment:
Corepack doesn't "just work" now?

Reply:
not available, because we don't create a wrapper for it on containerbase:

![image](https://github.com/renovatebot/renovate/assets/1798109/62343048-067d-4d26-901a-51f5c39b0cef)